segments/tests/test_models.py

```diff
@@ -32,11 +37,6 @@ def test_user_belongs_to_segment(self):
         s.refresh()
         self.assertTrue(s.has_member(self.u))
 
-    def test_user_adds_to_segment(self):
-        s = SegmentFactory()
-        s.add_member(self.u)
-        self.assertTrue(s.has_member(self.u))
```

Comment:
Duplicate of this one https://github.com/groveco/django-segments/blob/master/segments/tests/test_helpers.py#L24